examples/notebooks/Light_sources_example.ipynb

```diff
@@ -0,0 +1,151 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Solcore - Light sources examples\n",
+    "***\n",
+    "Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from solcore.light_source import LightSource"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Define the wavelength `wl` range of the spectra"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wl = np.linspace(300, 3000, 200)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now different types of light sources can be defined\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gauss = LightSource(source_type='laser', x=wl, center=800, linewidth=50, power=200)\n",
+    "bb = LightSource(source_type='black body', x=wl, T=5800, entendue='Sun')\n",
+    "am15g = LightSource(source_type='standard', x=wl, version='AM1.5g')\n",
+    "spectral = LightSource(source_type='SPECTRAL2', x=wl)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plot comparing the different light sources"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(1)\n",
+    "plt.plot(*gauss.spectrum(), label='Gauss')\n",
+    "plt.plot(*bb.spectrum(), label='Black body')\n",
+    "plt.plot(*am15g.spectrum(), label='AM1.5G')\n",
+    "plt.plot(*spectral.spectrum(), label='SPECTRAL2')\n",
+    "\n",
+    "plt.xlim(300, 3000)\n",
+    "plt.xlabel('Wavelength (nm)')\n",
+    "plt.ylabel('Power density (Wm$^{-2}$nm$^{-1}$)')\n",
+    "plt.tight_layout()\n",
+    "plt.legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plot comparing the spectra calculated with GAUSS at different hours of the day"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(2)\n",
+    "for h in range(8, 20):\n",
+    "    plt.plot(*gauss.spectrum(HOUR=h), label='{} h'.format(h))\n",
```

Comment:
The `gauss` type of spectrum does not have any time or geographical dependence. It's just a gaussian peak. If you don't want to use `smarts`, which is what the original example had (and I understand why you might not want to) then use `spectral`, which does have a time dependence. It should be something like (I haven't tested myself!):

```python
plt.plot(*spectral.spectrum(dateAndTime=datetime.datetime(2011, 6, 30, h), label='{} h'.format(h))
```
[`datetime`](https://docs.python.org/3.7/library/datetime.html) needs to be imported above for this to work.

Reply:
@dalonsoa thank's for the feedback! i changed from gaussian to spectral, let me know if there are any changes needed